src/components/content.js

```diff
@@ -698,15 +699,27 @@ class Content extends React.Component {
     if (this.props.readOnly) return
     if (!this.isInEditor(event.target)) return
 
-    event.preventDefault()
-    const data = getTransferData(event.clipboardData)
+    const handleData = (data) => {
```

Comment:
Actually we could probably have the callback just return `html`, and if it's null then have the logic call handled in the component layer. So maybe something along the lines of:

```
getHtmlFromNativePaste(this, (html) => {
  ...
})
```

But even that makes me wonder if we shouldn't be using `event.target` instead of passing around `this`.